pkgs/top-level/qt5-packages.nix

```diff
@@ -10,11 +10,15 @@
 , __splicedPackages
 , makeScopeWithSplicing'
 , generateSplicesForMkScope
+, pkgsBuildHost
 }:
 
 let
   pkgs = __splicedPackages;
-  qt5 = __splicedPackages.qt5;
+  # qt5 set should not be pre-spliced to prevent spliced packages being a part of an unspliced set
+  # 'pkgsCross.aarch64-multiplatform.pkgsBuildTarget.targetPackages.libsForQt5.qtbase' should not have a `__spliced` but if qt5 is pre-spliced then it will have one.
+  # pkgsBuildHost == pkgs
+  qt5 = pkgsBuildHost.qt5;
```

Comment:
Used pkgsBuildHost here while i should have used pkgsHostTarget

Reply:
https://github.com/NixOS/nixpkgs/pull/267503

Reply:
Yep, that does it.